### PR TITLE
Update Exchange-Teams-interact.md

### PR DESCRIPTION
**Pull request by dariomws**
Replace https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/5655.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -89,28 +89,24 @@ If users want the capability to schedule a Teams meeting using Exchange Server o
...
-- Both delegate and delegator must have a mailbox on the Exchange Server.
-
 - You must also complete steps 2-3 as described in [Configure Integration and OAuth between Skype for Business Online and Exchange Server](https://docs.microsoft.com/skypeforbusiness/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises); these steps will provide the Teams scheduling application the required permissions to confirm delegate permissions.
  
-> [!NOTE]
-> Step 2 includes role assignment for ArchiveApplication, which is not required for delegation.
+ > [!NOTE]
+ > Step 2 includes role assignment for ArchiveApplication, which is not required for delegation.
+
+- Teams Scheduling add-in for Outlook when scheduling a meeting on behalf of someone, requires Exchange 2013 CU19 or later. This is to support the unauthenticated discovery of the mailbox by our service to check delegate permissions against the delegator mailbox. The delegate and delegator location could be Exchange 2013 or later, or Exchange online, but Autodiscover must resolve to Exchange 2013 CU19 or later.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:109]:
```suggestion
- Teams Scheduling add-in for Outlook when scheduling a meeting on behalf of someone requires Exchange 2013 CU19 or later. This is to support the unauthenticated discovery of the mailbox by our service to check delegate permissions against the delegator mailbox. The delegate and delegator location could be Exchange 2013 or later, or Exchange online, but Autodiscover must resolve to Exchange 2013 CU19 or later.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**